test/test_autograd.py

```diff
@@ -8116,6 +8119,7 @@ def backward(ctx, x):
         c = Func.apply(a)
         self.assertEqual(repr(c), "tensor([2.], grad_fn=<FuncBackward>)")
 
+    @unittest.skipIf(IS_S390X, "Fails on s390x CI")
```

Comment:
You've introduced skipIfS390, why not use it?

Reply:
skipIfS390X is used only in one place, only because I haven't found how to use IS_S390X there. Everywhere else IS_* is used, so I'm trying to stick to more used solution for skipping.